app/client/ui/DocTour.ts

```diff
@@ -10,6 +10,10 @@ import {IconList, IconName} from 'app/client/ui2018/IconList';
 import {DocData} from 'app/common/DocData';
 import {dom} from 'grainjs';
 import sortBy = require('lodash/sortBy');
+import {marked} from "marked";
+import {renderer} from 'app/client/ui/DocTutorialRenderer';
+import {sanitizeTutorialHTML} from "./sanitizeHTML";
```

Comment:
```suggestion
import {sanitizeHTML} from "app/client/ui/sanitizeHTML";
```

You can use this variant instead, unless you need to tweak the default sanitization behavior, in which case you can just add a new variant like `sanitizeTourHTML`.

If no tweaking is necessary, you can just use the `markdown` helper from `app/client/lib/markdown.ts`.

Minor lint: absolute module path is preferred over relative.